tasks/release.js

```diff
@@ -0,0 +1,64 @@
+try {
+    require( 'shelljs/global' );
+    const colors = require( 'colors' );
+    const confirm = require( 'confirm-simple' );
+    const archiver = require( 'archiver' );
+    const fs = require( 'fs' );
+} catch( e ) {
```

Comment:
Simply not having a this try-catch statement will cause node to throw something like:`Cannot find module 'confirm-simple'`, which seems like a more useful message than this.